umbraco-cms-webapp-redis-cache/azuredeploy.parameters.json

```diff
@@ -1,57 +1,63 @@
 {
-   "$schema":"http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
-   "contentVersion":"1.0.0.0",
-   "parameters":{
-      "siteName":{
-         "value":"ExampleUmbracoApp"
-      },
-      "storageAccountType":{
-         "value":"Standard-GRS"
-      },
-      "redisCacheSKU":{
-         "value":"Standard"
-      },
-      "redisCacheCapacity":{
-         "value":0
-      },
-      "hostingPlanName":{
-         "value":"myappserviceplan"
-      },
-      "administratorLogin":{
-         "value":"umbracoadmin"
-      },
-      "administratorLoginPassword":{
-         "value":"pa$$word!"
-      },
-      "storageName":{
-         "value":"umbracomedia"
-      },
-      "redisCacheName":{
-         "value":"umbracosessioncache"
-      },
-      "nonAdminDatabaseUsername":{
-         "value":"umbracouser"
-      },
-      "nonAdminDatabasePassword":{
-         "value":"pa$$word!"
-      },
-      "edition":{
-         "value":"Standard"
-      },
-      "siteLocation":{
-         "value":"West US"
-      },
-      "serverName":{
-         "value":"NewServer"
-      },
-      "serverLocation":{
-         "value":"West US"
-      },
-      "redisLocation":{
-         "value":"West US"
-      },
-      "databaseName":{
-         "value":"ExampleDatabase"
-      }
-   }
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "deploymentLocation": {
```

Comment:
See https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#travis-ci for placeholder parameters for our CI